client/my-sites/checkout/cart/cart-coupon.jsx

```diff
@@ -102,7 +102,11 @@ class CartCoupon extends React.Component {
 					onChange={ this.handleCouponInput }
 					value={ this.state.couponInputValue }
 				/>
-				<button type="submit" className="button">
+				<button
+					type="submit"
+					className="cart__button button"
```

Comment:
Should there be any CSS updates that match the new `className` here? Should we also remove the `button` class?

Reply:
I didn't want to go out of scope there.

To be honest, I only had to add `cart_button` there because the error was on the same line of the actual change, breaking the commit hooks.
Funnily enough, by adding it, the line becomes too long and Prettier breaks it, moving it into its own line. 😅

Reply:
The funny thing is - that className should not be required at all. Instead, a `Button` component should be used (which has the `button` style applied internally). Not sure why it was done this way originally 🤔 
```
<Button disabled={} ></Button>
```
seems like a simpler fix than adding a custom class name that is not used anywhere :)